tiledb/sm/group/group.h

```diff
@@ -433,6 +454,39 @@ class Group {
    * Load group metadata, handles remote groups vs non-remote groups
    */
   void load_metadata();
+
+  /**
+   * Load group metadata from disk
+   */
+  void load_metadata_from_storage(
+      const shared_ptr<GroupDirectory>& group_dir,
+      const EncryptionKey& encryption_key,
+      Metadata* metadata);
+
+  /** Opens an group for reads. */
+  void group_open_for_reads();
+
+  /**
+   * Load group details from disk
+   */
+  void load_group_details();
+
+  /**
+   * Load a group detail from URI
+   *
+   * @param uri location to load
+   */
+  void load_group_from_uri(const URI& uri);
+
+  /**
+   * Load a group detail from URIs
+   *
+   * @param uris locations to load
+   */
+  void load_group_from_all_uris(const std::vector<TimestampedURI>& uris);
+
+  /** Opens an group for writes. */
+  void group_open_for_writes();
```

Comment:
Maybe these functions should return something.

Reply:
Not in scope for this PR, not even a little bit.